rust/scx_rustland_core/assets/bpf/main.bpf.c

```diff
@@ -577,7 +578,8 @@ static void dispatch_task(const struct dispatched_task_ctx *task)
 	if (!bpf_cpumask_test_cpu(task->cpu, p->cpus_ptr)) {
 		scx_bpf_dsq_insert_vtime(p, SHARED_DSQ, task->slice_ns, task->vtime, task->flags);
 		__sync_fetch_and_add(&nr_bounce_dispatches, 1);
-		goto out_kick_idle_cpu;
+		scx_bpf_kick_cpu(scx_bpf_task_cpu(p), SCX_KICK_IDLE);
```

Comment:
nit: can `task->cpu` be used here since it's already being used on the check in the outer `if` statement?

Reply:
We could, but the task won't be able to run on task->cpu because it's not in its allowed cpumask, so kicking that CPU isn't really useful

Reply:
I'm blind sorry about that